src/app/Library/CrudPanel/Traits/Relationships.php

```diff
@@ -57,16 +57,16 @@ public function inferRelationTypeFromRelationship($field)
         return Arr::last(explode('\\', get_class($relation)));
     }
 
-    public function getOnlyRelationEntity($relation_field)
+    public function getOnlyRelationEntity($field)
```

Comment:
@pxpm heads-up: I've taken the liberty to rename the variables here and use the Fluent string helper, no changes in behavior.